lib/pluck_each.rb

```diff
@@ -16,10 +16,13 @@ def pluck_each(*args)
 
     def pluck_in_batches(*column_names)
       options = column_names.extract_options!
+      string_column_names = column_names.map(&:to_s)
```

Comment:
Since `pluck` already [returns a flattened](https://github.com/rails/rails/blob/a678c47f6d31977e2a91cc198f115c803f0fc8e1/activesupport/lib/active_support/core_ext/enumerable.rb#L84-L88) response, I don't think we need to know if the original request had a single field. If we move the `flatten!` call into the `unless` block, we can make the stringification here more efficient as well.